dependencies.yaml

```diff
@@ -680,6 +804,14 @@ dependencies:
           - pytest<8
           - pytest-cov
           - pytest-xdist
+  test_python_pylibcudf:
```

Comment:
this should be added to the `all:` block above, along with other sets of testing dependencies

https://github.com/rapidsai/cudf/blob/f4a9b1c5016e254ebf2de55ac9946af6420ebff5/dependencies.yaml#L28-L30

That'll ensure that `pylibcudf`-specific testing dependencies:

* make it into the environment files in `conda/environments/` here
* are included in the environment that is created in devcontainers by `rapids-make-pip-env` and similar

Reply:
Thanks, added.